#!/usr/bin/env python
# dynsimpletrajectory.cfg
"""Dynamic reconfigure config file."""
PACKAGE = "simple_trajectory"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("trajectory_inflate", int_t, 0, "Trajectory Inflate [cells]", 5, 0, 100)
gen.add("map_inflate", int_t, 0, "Map Inflate [cells]", 5, 0, 100)
gen.add("publish_cropped_map", bool_t, 0, "Publish internal cropped map", False)

exit(gen.generate(PACKAGE, "simple_trajectory", "dynsimpletrajectory"))
